gbuzz-words.js

```diff
@@ -0,0 +1,22 @@
+var phrases = [
+  'big data',
+  'blockchain',
+  'bimodal',
+  'smart nation',
+  'analytics',
+  'cloud',
+  'machine learning',
+  'innovation',
+  'new technologies',
+  'agile',
+  'API',
+  'digitisation',
```

Comment:
```suggestion
  'digitalisation',
```

Reply:
hmm. I was wondering which of the 2 would be more likely to be used 🤣